### ISSUE TITLE
nuget.org naming

### ISSUE THREAD

**Issue by kalatchev** [lepoco/wpfui#1066]
### Is your feature request related to a problem? Please describe

Thanks for the great job!

I would like to suggest to rename nuget package like `lepoco.wpf-ui` and make `lepoco` as registered prefix. It takes minutes and looks far better, not to mention packages will be more distinguish. 

### Describe the solution you'd like

Register your company's name as nuget prefix.

### Describe alternatives you've considered

*

**Comment by syntax-tm** [comment]
Agreed. It makes finding the right NuGet package(s) that much easier. Wanted to include a screenshot of the badge that is displayed on NuGet.org:

![image](https://github.com/lepoco/wpfui/assets/13120514/8802eeff-94f0-48e3-a967-54bfeeb272f3)

And this is how it looks in VS:

![image](https://github.com/lepoco/wpfui/assets/13120514/cfe1cc21-ad79-46c1-aa9c-13225e0ab5b5)

For convenience you can review the id prefix reservation criteria [here](https://learn.microsoft.com/en-us/nuget/nuget-org/id-prefix-reservation#id-prefix-reservation-criteria).

**Comment by pomianowski** [comment]
Not planned for now.
